local-cli/templates/HelloWorld/android/build.gradle

```diff
@@ -3,6 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Are you sure this won't need to be updated to `classpath 'com.android.tools.build:gradle:3.1.3'`?

Reply:
Which I guess would require this change too: https://github.com/react-native-community/react-native-svg/issues/584#issuecomment-359261019

Reply:
But I'm not entirely sure, so yeah just asking

Reply:
nvm I saw you did a dedicated PR for that https://github.com/facebook/react-native/pull/19697

Reply:
This patch just works for me, you need to add the changes to build.gradle file in the root of a project. Please double check

Bumping gradle plugin to 3.x will cause many issues, like requires build-tools 27

Reply:
awesome, thanks for clarifying, as soon as we have a new RC out I'll test and report back :)